packages/core/src/testUtils.tsx

```diff
@@ -125,7 +125,11 @@ export const mockImplicitAccount = (
   }
 };
 
-export const mockMnemonicAccount = (index: number, label?: string): MnemonicAccount => ({
+export const mockMnemonicAccount = (
+  index: number,
+  label?: string,
+  isVerified: boolean = true
```

Comment:
nit: boolean is redundant

not-nit: please make it accept Partial<MnemonicAccount> instead, to avoid things like
```{...mockMnemonicAccount(0), isVerified: false}```